101-workshop-nodejs/ex4-exposing-apis-from-actions/concepts.md

```diff
@@ -19,30 +19,27 @@
 
 # Concepts
 
-## Web Actions
+## Web actions
 
-Cloud Function actions can be annotated with a special flag (i.e., `--web true`) at creation to convert them into "web actions". The result is the corresponding creation of a public URL that can be used to trigger the action from any web app.
+Cloud Function actions can be annotated with a special flag, such as`--web true`, at creation to convert them into [web actions](https://cloud.ibm.com/docs/openwhisk?topic=cloud-functions-actions_web). The result is the corresponding creation of a public URL that can be used to trigger the action from any web app.
```

Comment:
It is not "such as" it "is" (only) `--web true`

Reply:
Then can it just read like: "Cloud Function actions can be annotated with a special flag,`--web true`, at creation to.."

Reply:
works nicely